python/tests/experimental/performance_estimation/test_performance_estimation.py

```diff
@@ -0,0 +1,116 @@
+import pandas as pd
+import pytest
+
+import whylogs as why
+from whylogs.core.schema import DatasetSchema
+from whylogs.core.segmentation_partition import (
+    ColumnMapperFunction,
+    SegmentationPartition,
+    segment_on_column,
+)
+from whylogs.experimental.performance_estimation import AccuracyEstimator
+
+
+@pytest.fixture
+def acc_estimator(reference_df):
+    reference_result = make_result_set(reference_df)
+    acc_estimator = AccuracyEstimator(reference_result)
+    return acc_estimator
+
+
+@pytest.fixture
+def reference_df():
+    data = {
+        "product": ["milk", "carrot", "cheese", "broccoli"],
+        "category": ["dairies", "vegetables", "dairies", "vegetables"],
```

Comment:
minor: "dairy" is applies to plural items as a category label, rename from "dairies"

Reply:
ok, will do in a follow-up